src/lib/layout-utils.ts

```diff
@@ -58,7 +58,7 @@ export function calcBottomStatusesAreaLpx(
     bottomStatusCount: number,
 ): Rect {
     return {
-        x: chartSettings.legend.y.width,
+        x: chartSettings.legend.y.width ,
```

Comment:
We should probably configure prettier in pre-commit for this project ;)

Reply:
Yep, fixed